packages/strapi-plugin-i18n/admin/src/components/LocalePicker/index.js

```diff
@@ -3,6 +3,7 @@ import { useSelector } from 'react-redux';
 import { Picker, Padded, Text, Flex } from '@buffetjs/core';
 import { Carret } from 'strapi-helper-plugin';
 import styled from 'styled-components';
+import get from 'lodash/get';
```

Comment:
`import { get } from 'lodash';`

Reply:
It treeshakes better when imported using import get from 'lodash/get';